api/turing/api/appcontext.go

```diff
@@ -62,18 +61,17 @@ func NewAppContext(
 		return nil, errors.Wrapf(err, "Failed initializing MLP Service")
 	}
 
-	// Create a map of env name to cluster name for each supported deployment environment
-	envClusterMap, err := getEnvironmentClusterMap(mlpSvc, []string{cfg.EnsemblerServiceBuilderConfig.ClusterName})
+	envClusterMap, err := buildKubeconfigStore(mlpSvc, cfg)
 	if err != nil {
-		return nil, errors.Wrapf(err, "Error obtaining environment info from MLP Service")
+		return nil, errors.Wrapf(err, "Error obtaining environment info from MLP Service and constructing kubeconfig store")
```

Comment:
Do we still obtain something from MLP, or do we rely on file only and mention MLP can be removed?

Reply:
Yes, the list of environments is obtained from mlp first, before checking if there are k8sconfigs provided for those environments.